### PR TITLE
Added getinfo endpoint

### PR DESCRIPTION
**Pull request by momboto**

### CODE REVIEW

File: kount_access/access_sdk.py [hunk 1]
```diff
@@ -334,6 +339,87 @@ def get_devices(self, uniq):
...
+        data = {'v': self.version, 's': session,
+                'i': self._calc_data_set_value(info, velocity, decision, trusted, behaviosec)}
+
+        if trusted or behaviosec:
+            self._validate_param(uniq, "invalid uniq: ")
+            data['uniq'] = uniq
+
+        if behaviosec:
+            data['m'] = self.merchant_id
+            data['timing'] = "dasda"
```
Comment (boyankukushev) [inline kount_access/access_sdk.py]: I don't think this should be `dasda` ..

File: kount_access/access_sdk.py [hunk 2]
```diff
@@ -334,6 +339,87 @@ def get_devices(self, uniq):
...
+            i += self.DATA_SET['velocity']
+        if decision:
+            i += self.DATA_SET['decision']
+        if trusted:
+            i += self.DATA_SET['trusted']
+        if behaviosec:
+            i += self.DATA_SET['behaviosec']
+        return i
+
+    def _prepare_hashes(self, params, username, password):
```
Comment (boyankukushev) [inline kount_access/access_sdk.py]: is this function used at other places in the sdk?

File: kount_access/access_sdk.py (hunk 1 above)
Comment (momboto) [reply kount_access/access_sdk.py]: data['timing] not used in info endpoint. I'll remove it. Thanks.

File: kount_access/access_sdk.py (hunk 2 above)
Comment (momboto) [reply kount_access/access_sdk.py]: Yes, it used in _prepare_params and get_info methods

**Pull request merged**